src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,9 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+   const employeeInformation = employees.find(employee => {
+    return employee.name == employeeName;
+   })
+   const employeeRole = employeeInformation.role;
+   return employeeRole;
+};
```

Comment:
Great solution, and it can be refactored into one line :)

First of all you could merge line 5 and line 6 into:
`````
return employeeInformation.role;
``````
And this would be already nice but we can go one step further and add the .role selector at the end of the find method. In this case we need to rename our employeeInformation and choose a better more descriptive word, like employeeRole
`````
  const employeeRole = employees.find((employee) => {
    return employee.name == employeeName;
  }).role;
  return employeeRole;
`````
At this point we can even go a little bit further to return the method directly : 
`````
const getEmployerRole = (employeeName, employees) => {
  return employees.find((employee) => {
    return employee.name == employeeName;
  }).role;
};
``````
And then this could be written in one line such as:

`````
const getEmployerRole = (employeeName, employees) =>
  employees.find((employee) => employee.name == employeeName).role;
`````
And at this point esLint is complaining about using == instead of ===.
As standard always use the === (strictly equal operator) because we want the type and the value equal as well.